### PR TITLE
Bumped version to 6.1.1

### PR DESCRIPTION
**Pull request by MEseifan**

### CODE REVIEW

**Comment by dereklwood** [comment]
We should be merging this into branch release/6.1.

Also, the CSD versioning scheme is that CSD version x.y.* is compatible with CDAP x.y.* (basically patch versions are independent), so technically this CSD release should be version 6.1.0.

One last thing is that we should update the changelog before release as well

File: src/descriptor/service.sdl [hunk 1]
```diff
@@ -2013,6 +2022,17 @@
       "label": "CDAP Gateway/Router Service",
       "pluralLabel": "CDAP Gateway/Router Services",
       "parameters": [
+        {
+          "name": "router_ssl_cert_path",
+          "label": "Router SSL Cert Path",
+          "description": " File path to certificate and private key file in PEM format to use for CDAP Router
```
Comment (dereklwood) [inline src/descriptor/service.sdl]: leading whitespace. Also (for better or worse) we have kept descriptions single-line.

File: src/descriptor/service.sdl [hunk 2]
```diff
@@ -2369,6 +2389,15 @@
           "type": "long",
           "max": 65535
         },
+        {
+          "name": "network_proxy_address",
+          "label": "Network Proxy Address",
+          "description": "  A SOCKS 5 proxy address for CDAP to proxy outgoing connections.
```
Comment (dereklwood) [inline src/descriptor/service.sdl]: trim leading whitespace

File: src/descriptor/service.sdl [hunk 3]
```diff
@@ -2013,6 +2022,17 @@
       "label": "CDAP Gateway/Router Service",
       "pluralLabel": "CDAP Gateway/Router Services",
       "parameters": [
+        {
+          "name": "router_ssl_cert_path",
```
Comment (dereklwood) [inline src/descriptor/service.sdl]: we may need to discuss with whoever added this option, how this relates to the existing keystore paths.  CM provides [additional integration](https://github.com/cdapio/cm_csd/blob/develop/src/descriptor/service.sdl#L2154) for SSL which we have already configured, and I don't know if this new option can play nicely with it.

Comment (dereklwood) [reply src/descriptor/service.sdl]: after discussion, let's remove this entry, as CMs sslServer configuration will take precedence over it anyway

File: src/descriptor/service.sdl [hunk 4]
```diff
@@ -1659,6 +1659,15 @@
       "default":false,
       "type": "boolean"
     },
+    {
+      "name": "ssl_internal_cert_path",
+      "label": "SSL Internal Cert PAth",
```
[the remark this answers is not in the document]
Comment (MEseifan) [reply src/descriptor/service.sdl]: done

File: src/descriptor/service.sdl (hunk 2 above)
Comment (MEseifan) [reply src/descriptor/service.sdl]: done

File: src/descriptor/service.sdl (hunk 3 above)
Comment (MEseifan) [reply src/descriptor/service.sdl]: done

File: src/descriptor/service.sdl [hunk 5]
```diff
@@ -2,7 +2,7 @@
   "name": "CDAP",
   "label": "CDAP",
   "description": "The Cask Data Application Platform (CDAP) is an easy-to-use, open source and enterprise-ready integrated platform for organizations to build, deploy, and operate data-driven applications.",
-  "version": "6.1.0-SNAPSHOT",
+  "version": "6.1.1",
```
Comment (dereklwood) [inline src/descriptor/service.sdl]: 6.1.0 to match above

File: src/descriptor/service.sdl [hunk 6]
```diff
@@ -2369,6 +2379,15 @@
           "type": "long",
           "max": 65535
         },
+        {
+          "name": "network_proxy_address",
+          "label": "Network Proxy Address",
+          "description": "A SOCKS 5 proxy address for CDAP to proxy outgoing connections.
+      The address must be in the format of "host:port".",
```
Comment (dereklwood) [inline src/descriptor/service.sdl]: consolidate lines, and i think the extra quotes causes this to be invalid JSON

File: src/descriptor/service.sdl [hunk 7]
```diff
@@ -2013,6 +2022,7 @@
       "label": "CDAP Gateway/Router Service",
       "pluralLabel": "CDAP Gateway/Router Services",
       "parameters": [
+
```
Comment (dereklwood) [inline src/descriptor/service.sdl]: remove blank line

File: src/descriptor/service.sdl (hunk 5 above)
Comment (MEseifan) [reply src/descriptor/service.sdl]: done

File: src/descriptor/service.sdl (hunk 7 above)
Comment (MEseifan) [reply src/descriptor/service.sdl]: done

File: src/descriptor/service.sdl (hunk 6 above)
Comment (MEseifan) [reply src/descriptor/service.sdl]: done

**Pull request merged**